app/Controllers/BaseController.php

```diff
@@ -37,6 +37,12 @@ abstract class BaseController extends Controller
      */
     protected $helpers = [];
 
+    /**
+     * Be sure to declare properties for anything you assign.
```

Comment:
```suggestion
     * Be sure to declare properties for any property fetch you initialized in initController method.
```

Reply:
It is not only in `initController()` method, but also in all methods.

Reply:
I see, updated the suggestion 👍